src/widgets/grid.ts

```diff
@@ -1,9 +1,15 @@
+import { ElementArrayFinder, ElementFinder } from "../wdio";
 import { ContextMenu } from './context-menu';
 import { Widget } from './widget';
-import { ElementArrayFinder, ElementFinder } from "../wdio";
 
 
 export class Grid extends Widget {
+    public async waitUntilLoaded(cssTag: string , timeout: number = 5000) {
```

Comment:
Can't we set a default value for the `cssTag` argument? If I remember correctly, grids have a default "Loading" overlay that can be customized. Thus, I would suggest setting this argument to a default value that matches with a selector for this "Loading" overlay

Reply:
We can use ".ag-overlay-loading-wrapper" as default cssTag if you all agree.